mitosheet/src/mito/components/toolbar/GraphTabs/ChartFormatTabContents.tsx

```diff
@@ -0,0 +1,99 @@
+        analysisData: AnalysisData;
+        sheetDataArray: SheetData[];
+        selectedColumnsIds?: ColumnID[];
+        mitoContainerRef: React.RefObject<HTMLDivElement>;
+    }): JSX.Element => {
+    const currOpenTaskpane = props.uiState.currOpenTaskpane;
+    if (currOpenTaskpane.type !== TaskpaneType.GRAPH) {
+        throw new Error('ChartDesignTabContents should only be used when the graph taskpane is open.');
+    }
+    const openGraph = currOpenTaskpane.openGraph;
+    
+    // Get the step params and the step id of the currently open graph by getting the most recent step from this array
+    // This is used in conjunction with useLiveUpdatingParams in the GraphSidebar with a special flag to add these edits
+    // into the same step as useLiveUpdatingParams is using. This assumes that there is already a step in the stepSummaryList
+    // for this graph.
+    const stepSummaryList = props.analysisData.stepSummaryList;
+    const currGraphStep = stepSummaryList[stepSummaryList.length - 1];
+    const params = currGraphStep.params as GraphParamsBackend | undefined;
+
+    // When edits happen, just call the API directly w/ the graph id and the step id
+    const updateGraphParam = (update: RecursivePartial<GraphParamsBackend>): void => {
+        if (params === undefined) {
+            return;
+        }
+        void props.mitoAPI.editGraph(
+            openGraph.graphID,
+            convertBackendtoFrontendGraphParams(updateObjectWithPartialObject(params, update)),
+            params.graph_rendering.height ?? '100%',
+            params.graph_rendering.width ?? '100%',
+            currGraphStep.step_id,
+            true
+        );
+    }
+
+    const elementOptions = ['Chart Title', 'Plot', 'Legend', 'Gridlines', 'Axes', 'Facet'];
+    const [currElement, setCurrElement] = React.useState<typeof elementOptions[number]>('Chart Title');
+
+    return (<div className='mito-toolbar-bottom'>
+        <Select
+            display={props.uiState.currOpenDropdown === 'chart-format'}
+            width='small'
+            onChangeDisplay={(display) => {
+                props.setUIState(prevUIState => {
+                    return {
+                        ...prevUIState,
+                        currOpenDropdown: display ? 'chart-format' : undefined
+                    }
+                })
+            }}
+            onChange={(value) => {
+                setCurrElement(value as typeof elementOptions[number]);
+            }}
+            value={currElement}
+        >
+            { elementOptions.map((element, index) => 
+                (<DropdownItem
+                    key={index}
+                    onClick={() => {
+                        setCurrElement(element);
+                    }}
+                    title={element}
+                />)
+            ) }
+        </Select>
+        {currElement === 'Chart Title' && <ChartTitleFormatOptions params={params} updateGraphParam={updateGraphParam}/>}
```

Comment:
Thoughts on defining a `ChartFormatItem` type and then using a switch statement here so we can be type safe if we ever change the titles?

Reply:
Oh yeah I'm actually auto-generating it based off of the `elementOptions` array

Reply:
I could add a switch but I think it'd actually be more lines of code

Reply:
Sweet sounds good